### PR TITLE
Make fails on OS X due to sed differences. 

### PR DESCRIPTION
**Pull request by ankur325**
Added support for both OS X and Linux.

### CODE REVIEW

**Review by hawk248 — APPROVED** [review]

File: integration_tests/ci/run-all.sh [hunk 1]
```diff
@@ -106,10 +111,10 @@ stop_rest_server() {
 set -euo pipefail
 
 log "Compiling local binaries"
-make install &>${DETAILED_OUTPUT_TARGET}
+#make install &>${DETAILED_OUTPUT_TARGET}
```
Comment (ashcherbakov) [inline integration_tests/ci/run-all.sh:114]: Is it commented intentionally?

**Comment by ashcherbakov** [comment]
@ankur325 The integration tests failed for this PR. Can you please have a look?

File: integration_tests/ci/run-all.sh (hunk 1 above)
Comment (ankur325) [reply integration_tests/ci/run-all.sh]: @ashcherbakov - Commented by mistake, I was re-running the tests on local so commented for speed.

**Comment by ankur325** [comment]
> @ankur325 The integration tests failed for this PR. Can you please have a look?

@ashcherbakov  Can you re-run the workflows. It is passing on my local.

**Review by ashcherbakov — APPROVED** [review]

**Pull request merged**